src/test/ui/in-band-lifetimes/no_in_band_in_struct.stderr

```diff
@@ -0,0 +1,14 @@
+error[E0261]: use of undeclared lifetime name `'test`
+  --> $DIR/no_in_band_in_struct.rs:15:9
+   |
+15 |     x: &'test u32,
+   |         ^^^^^ undeclared lifetime
```

Comment:
I wonder if we should give some better tip here. Seems orthogonal to your PR, but might be worth opening a follow-up pointing to the struct and suggesting that the lifetime parameter be added there.

Reply:
https://github.com/rust-lang/rust/issues/46135